swagger.yaml

```diff
@@ -1227,6 +1228,12 @@ definitions:
       ssh_public_key:
         type: string
         description: SSH public key for debugging OpenShift nodes.
+      dhcp_enabled:
```

Comment:
I would use a more descriptive name.  Maybe vip_auto_allocation?  Same for the description.